cluster/examples/kubernetes/ceph/operator-openshift.yaml

```diff
@@ -269,6 +269,21 @@ spec:
         #  value: "9080"
         - name: ROOK_HOSTPATH_REQUIRES_PRIVILEGED
           value: "true"
+
+        # Time to wait until the node controller move Rook pods to other nodes
+        # after detecting a unreachable node.
+        # Pods affected by this setting are:
+        # mgr, rbd, mds, rgw, PVC based mons and osds, and ceph toolbox
+        # The value used in this variable replaces the default value of 300 secs
+        # added automatically by k8 as Toleration for
+        # <node.kubernetes.io/unreachable>
+        # The total amount of time to reschedule Rook pods in healthy nodes
+        # before detecting a <not ready node> condition will be the sum of:
+        #  --> node-monitor-grace-period: 40 seconds (node controller flag)
```

Comment:
Indicate that this is a k8s flag and to what k8s component it belongs.

Reply:
Done.